pipelines/datasets/br_inmet_bdmep/utils.py

```
@@ -5,26 +5,31 @@
 
-from pipelines.datasets.br_inmet_bdmep.constants import ConstantsMicrodados
+from pipelines.datasets.br_inmet_bdmep.constants import (
+    constants as constants_microdados,
+)
 from pipelines.utils.utils import log
 
 
 def get_latest_dowload_link() -> str:
     try:
-        reponse = requests.get(ConstantsMicrodados.URL.value)
+        reponse = requests.get(constants_microdados.URL.value)
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)